docs/README.md

```diff
@@ -0,0 +1,624 @@
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=2018-01,2018-03&group_by=date"
+# [
+#   {
+#     "date": "2018-01-01",
+#     "volume": 28352.73849
+#   },
+#   {
+#     "date": "2018-02-01",
+#     "volume": 27424.03418
+#   },
+#   {
+#     "date": "2018-03-01",
+#     "volume": 38738.38481
+#   }
+# ]
+
+# Now, let's see the volume for credit and sig debit:
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=date=2018-01;card=credit,sig_debit&group_by=card"
+# [
+#  {
+#    "card": "credit",
+#    "volume": 28327.3061
+#  },
+#  {
+#    "card": "sig_debit",
+#    "volume": 1924.48568
+#  }
+#]
+
+# How about focused on Omaha, Iowa, Kansas, and Missouri?
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=date=2018-01;card=credit,sig_debit;state=MO,KS,NE,IA&group_by=card"
+# [
+#   {
+#     "card": "credit",
+#     "volume": 23394.83581
+#   },
+#   {
+#     "card": "sig_debit",
+#     "volume": 1483.83589
+#   }
+# ]
+
+# Putting these examples together, one can see metrics per transaction in specific states grouped by card and date.
+curl -H "Authorization: Bearer $ID_TOKEN" \
+        "$BASE_URL/query?metrics=volume&filter=date=2018-01,2018-03;card=credit,sig_debit;state=MO,KS,NE,IA&group_by=date,card&normalization=transaction"
+# [
+#   {
+#     "card": "credit",
+#     "date": "2018-01-01",
+#     "volume": 53.77786
+#   },
+#   {
+#     "card": "sig_debit",
+#     "date": "2018-01-01",
+#     "volume": 8.38631
+#   },
+#   {
+#     "card": "credit",
+#     "date": "2018-02-01",
+#     "volume": 68.85441
+#   },
+#   {
+#     "card": "sig_debit",
+#     "date": "2018-02-01",
+#     "volume": 2.41721
+#   },
+#   {
+#     "card": "credit",
+#     "date": "2018-03-01",
+#     "volume": 58.09631
+#   },
+#   {
+#     "card": "sig_debit",
+#     "date": "2018-03-01",
+#     "volume": 5.42187
+#   }
+# ]
+```
+
+### Aggregation
+
+Abstract aggregation operation.
+    
+
+<details markdown='1'><summary>Aggregations</summary>
+
+#### None
+
+
+
+#### 3 Month Moving Average
+
+Periods = 3, Frequency = Month
+    
+
+#### 6 Month Moving Average
+
+Periods = 6, Frequency = Month
+    
+
+#### 12 Month Moving Average
+
+Periods = 12, Frequency = Month
+    
+
+#### 18 Month Moving Average
+
+Periods = 18, Frequency = Month
+    
+
+</details>
+
+### Attributes
+
+Attribute base class.
+Metrics are columns of interest (Ex. Volume, Net Revenue, Processing Cost) that are co-tabulated with
+attribute columns (Ex. Date, Card, Region) that are used to filter or group the metric.
+
+<details markdown='1'><summary>Attributes</summary>
+
+#### Card
+
+**Card** is an attribute of central importance in the AIM system.
+
+There are 5 basic card types:
+- credit
+- signature_debit aka *sig_debit*
+- pin_debit
+- opt_blue
```

Comment:
Al:
> Where is the 5th one and also do you want to mention what are the subsets of bankcards, credit, and
sig debit-(Mastercard, Visa, Discover), pin debit –(???), other-(???) and define opt blue? – Question
for Josh?

Reply:
Either:
- let's have the AIM team define an entirely new description for Card and the values
- give me the card components/definitions to add (preferably exact copy)